pkg/products/monitoring/prometheusRules.go

```diff
@@ -164,22 +164,22 @@ func (r *Reconciler) newAlertsReconciler() resources.AlertReconciler {
 						Labels: map[string]string{"severity": "warning"},
 					},
 					{
-						Alert: "PVCStorageWillFillIn4Days",
+						Alert: "KubePersistentVolumeFillingUp",
 						Annotations: map[string]string{
 							"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/pvc_storage.asciidoc#pvcstoragewillfillin4hours",
-							"message": "The {{  $labels.persistentvolumeclaim  }} PVC will run of disk space in the next 4 days",
+							"message": "Based on recent sampling, the PersistentVolume claimed by {{ $labels.persistentvolumeclaim }} in Namespace {{ $labels.namespace }} is expected to fill up within four days. Currently {{ $value | humanizePercentage }} is available.",
 						},
-						Expr:   intstr.FromString("(predict_linear(kubelet_volume_stats_available_bytes{job='kubelet'}[6h], 4 * 24 * 3600) <= 0 AND kubelet_volume_stats_available_bytes{job='kubelet'} / kubelet_volume_stats_capacity_bytes{job='kubelet'} < 0.25 )* on(namespace) group_left(label_monitoring_key) kube_namespace_labels{label_monitoring_key='middleware'}"),
+						Expr:   intstr.FromString("(predict_linear(kubelet_volume_stats_available_bytes{job='kubelet'}[6h], 4 * 24 * 3600) <= 0 and kubelet_volume_stats_available_bytes{job='kubelet'} / kubelet_volume_stats_capacity_bytes{job='kubelet'} < 0.15) * on(namespace) group_left(label_monitoring_key) kube_namespace_labels{label_monitoring_key='middleware'}"),
 						For:    "15m",
 						Labels: map[string]string{"severity": "warning"},
 					},
 					{
-						Alert: "PVCStorageWillFillIn4Hours",
+						Alert: "KubePersistentVolumeFillingUp",
 						Annotations: map[string]string{
 							"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/2.x/alerts/pvc_storage.asciidoc#pvcstoragewillfillin4hours",
```

Comment:
@grdryn Would it make sense to update the sop content as well with the new alert name(s)?
Perhaps have both alert names in the sop to allow for a transitionary period where it could be either

Reply:
I've created https://github.com/RHCloudServices/integreatly-help/pull/624 now with a new section in that SOP file, and updated the link here. Please take a look! (it's mostly a copy of the previous troubleshooting steps